server/app/views/v1/host_nodes/health.json.jbuilder

```diff
@@ -3,8 +3,14 @@ json.name @node.name
 json.node_number @node.node_number
 json.initial_member @node.initial_member?
 
+json.status @node.status
+json.connected_at @node.connected_at
+json.disconnected_at @node.disconnected_at
 json.connected @node.connected
+
 json.etcd_health do
-  json.health @etcd_health[:health]
-  json.error @etcd_health[:error]
+  json.health @node_health.dig(:etcd_health, :health)
+  json.error @node_health.dig(:etcd_health, :error)
```

Comment:
The agent can return an `{error: ...}`: https://github.com/kontena/kontena/blob/master/agent/lib/kontena/rpc/etcd_api.rb#L62

This kind of "agent got an etcd API error" is distinct from "server got an agent RPC error".